account/identity.go

```diff
@@ -81,7 +87,7 @@ func (m Identity) ConvertIdentityFromModel() *app.Identity {
 			Type: "identities",
 			Attributes: &app.IdentityDataAttributes{
 				Username: &m.Username,
-				Provider: &m.Provider,
+				Provider: &m.ProviderType,
```

Comment:
Shouldn't we also have to modify `IdentityDataAttributes` to rename `provider` to  `provider_type` ?

```
// identityDataAttributes represents an identified user object attributes
var identityDataAttributes = a.Type("IdentityDataAttributes", func() {
	a.Attribute("fullName", d.String, "The users full name")
	a.Attribute("imageURL", d.String, "The avatar image for the user")
	a.Attribute("username", d.String, "The username")
	a.Attribute("email", d.String, "The email")
	a.Attribute("bio", d.String, "The bio")
	a.Attribute("url", d.String, "The url")
	a.Attribute("provider", d.String, "The IDP provided this identity")
})
```